chart/values.yaml

```
@@ -234,6 +234,8 @@ mayastor:
       iface: ""
       # Reservations Persist Through Power Loss State
       ptpl: true
+  # Workaorund for mayastor io-engine on Intel Xeon E-2278G CPU @ 3.40GHz
+  envcontext: ""
```

I would probably be a bit specific, example:
```suggestion
  # Pass additional arguments to the Environment Abstraction Layer.
  # Example: workaround for running on Intel Xeon E-2278G CPU @ 3.40GHz: "iova-mode=pa"
  envcontext: ""
```